api/src/certification/flash-certification/domain/models/AssessmentSimulatorSingleMeasureStrategy.js

```diff
@@ -17,15 +17,15 @@ export class AssessmentSimulatorSingleMeasureStrategy {
     });
 
     const nextChallenge = this.pickChallenge({ possibleChallenges });
+    if (this.#hasNoNextChallenge({ nextChallenge })) {
```

Comment:
Je trouve l'abstraction un peu inutile et pas forcément simple à lire. (Bizarre de passer un challenge dans la fonction pour vérifier qu'il n'y pas de challenge 🤔 )

```suggestion
if (!nextChallenge)
```